network/backend/lnd/lnd.go

```diff
@@ -358,8 +419,8 @@ func (l Backend) GetChannelInfo(ctx context.Context, channel *models.Channel) er
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	if info != nil {
-		channel.WeFirst = resp.Node1Pub == info.IdentityPubkey
+	if info != nil && resp.Node1Pub != info.IdentityPubkey {
+		channel.Policy1, channel.Policy2 = channel.Policy2, channel.Policy1
```

Comment:
As mentioned in https://github.com/edouardparis/lntop/pull/86 we could rename `Policy1` to `LocalPolicy` and `Policy2` to `RemotePolicy` or something along these lines in order to make our code more readable.

Reply:
Yep, agree!